### DISCUSSION TITLE
WWE 2K18

### DISCUSSION THREAD

**Discussion by raviii12** [skyline-emu/skyline#524] [Game Requests]
Sir plzzz test wwe games

**Comment by PixelyIon** [comment]
Since you haven't specified a WWE game, I'm going to go ahead and assume you meant WWE 2K18.

**Reply by Irfankhan886** [reply]
Nsp faile dedo

**Reply by raviii12** [reply]
Yes wwe18 plzz make it playble on skyline emulator

**Comment by Irfankhan886** [comment]
Bhai WWE dedo
